pyproject.toml

```diff
@@ -107,6 +109,7 @@ doctest = "pytest -n auto --doctest-modules svgdigitizer"
 black = "*"
 isort = "*"
 pylint = "*"
+python = "3.14.*"
```

Comment:
I am not sure if this intended or necessary. It is already defined in the specifications for the environment:

```toml
[tool.pixi.feature.dev.dependencies]
pip = "*"
python = "3.14.*"
```